src/dygraph.js

```diff
@@ -861,7 +861,11 @@ Dygraph.prototype.resizeElements_ = function() {
   this.graphDiv.style.width = this.width_ + "px";
   this.graphDiv.style.height = this.height_ + "px";
 
-  var canvasScale = utils.getContextPixelRatio(this.canvas_ctx_);
+  var pixelRatioOption = this.getNumericOption('pixelRatio')
+
+  var canvasScale = pixelRatioOption > 0
```

Comment:
Any reason not to do `pixelRatioOption || utils.getContextPixelRatio(this.canvas_ctx_)`?

Reply:
Probably trying to make my linter happier with 80 char lines, I switched it back here and in range-selector.js, it does read more naturally.